src/bananabot.py

```diff
@@ -59,14 +61,20 @@ def start():
             workers[worker.id] = worker
             workers[worker.id].start()
 
+    return True
+
+# Terminates all workers
 def stop():
     for key in workers:
         workers[key].terminate()
     
     globals()['workers'] = {}
+    return True
```

Comment:
Ditto for the comment above

Reply:
See https://github.com/NickoOwen/bananabot/pull/18/files#r1083293567